lightning/src/chain/channelmonitor.rs

```diff
@@ -2531,9 +2581,11 @@ impl<Signer: WriteableEcdsaChannelSigner> ChannelMonitorImpl<Signer> {
 		ret
 	}
 
-	pub fn get_and_clear_pending_events(&mut self) -> Vec<Event> {
+	/// Gets the set of events which are repeated regularly (i.e. those which RBF bump
+	/// transactions). We're okay if we lose these on restart as they'll be regenerated for us at
+	/// some regular inverval.
+	pub(super) fn get_repeated_events(&mut self) -> Vec<Event> {
```

Comment:
The wording of this last sentence and possibly the method named can be improved here. Isn't _this_ method what is generating the events? Or is it referring to the `ClaimEvent` not the `Event`?